### ISSUE TITLE
ZRANGEBYSCORE .. WITHSCORES has return order reversed

### ISSUE THREAD

**Issue by mchang31415** [YugaByte/yugabyte-db#39]
YUGABYTE:
```
127.0.0.1:2017> ZADD MCCHANG_TEST_SET2 123 "asdf"
(integer) 1
127.0.0.1:2017> ZRANGEBYSCORE MCCHANG_TEST_SET2 -inf +inf WITHSCORES
1) "123.000000"
2) "asdf"
```

REDIS:
```
127.0.0.1:6379>  ZADD MCCHANG_TEST_SET2 123 "asdf"
(integer) 1
127.0.0.1:6379>  ZRANGEBYSCORE MCCHANG_TEST_SET2 -inf +inf WITHSCORES
1) "asdf"
2) "123"
```

Jedis expects the second value to be a double, and tries to parse as such, leading to exceptions as
```
Caused by: java.lang.NumberFormatException: For input string: "387a086f-6463-3b26-ae11-5fc6c54aaacf"
        at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
        at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.lang.Double.parseDouble(Double.java:538)
        at java.lang.Double.valueOf(Double.java:502)
        at redis.clients.jedis.BuilderFactory$15.build(BuilderFactory.java:267)
        at redis.clients.jedis.BuilderFactory$15.build(BuilderFactory.java:257)
        at redis.clients.jedis.Response.build(Response.java:61)
        at redis.clients.jedis.Response.get(Response.java:37)
       ...
```

**Comment by kmuthukk** [comment]
Thanks @mchang31415 for reporting this issue. We will address this quickly...

**Comment by kmuthukk** [comment]
@rahuldesirazu : could you link to the commit id for this fix, and close out the issue?

**Comment by rahuldesirazu** [comment]
Resolved by 1f2f38d4cd5096c38eb1652a1f4cdc58362dc5f0.
